src/tensor_ops/reshape_to/mod.rs

```diff
@@ -68,6 +68,23 @@ pub trait ReshapeTo: HasErr + HasShape {
     fn reshape_like<Dst: Shape>(self, dst: &Dst) -> Option<Self::WithShape<Dst>> {
         self.try_reshape_like(dst).map(Result::unwrap)
     }
+    /// Ensures the tensor's memory is contiguous.
+    ///
+    /// If the memory is already contiguous no copying is performed.
+    fn contiguous(self) -> Self::WithShape<Self::Shape>
+    where
+        Self::Shape: ConstShape,
+    {
+        self.try_contiguous().unwrap()
+    }
+    /// See [`ReshapeTo::contiguous`]
+    fn try_contiguous(self) -> Result<Self::WithShape<Self::Shape>, Self::Err>
+    where
+        Self::Shape: ConstShape,
```

Comment:
We actually don't need these bounds, mind removing it from both of the methods?
```suggestion
    {
        self.try_contiguous().unwrap()
    }
    /// See [`ReshapeTo::contiguous`]
    fn try_contiguous(self) -> Result<Self::WithShape<Self::Shape>, Self::Err>
```